app/app.js

```diff
@@ -5,6 +5,8 @@ import config from './config/environment';
 
 let App;
 
+window.EmberENV = window.EmberENV || {};
+window.EmberENV.ENABLE_DS_FILTER = true;
```

Comment:
@shivanibhanwal should this go in config/environments.js? I think it merges EmberENV config there.